docs/companion.md

```diff
@@ -643,10 +635,12 @@ risk.**
 
 :::
 
-#### `corsOrigins` `COMPANION_CLIENT_ORIGINS`
+#### `corsOrigins` `COMPANION_CLIENT_ORIGINS` (required)
```

Comment:
We discussed we want to write docs on the different supported values between `corsOrigins` and `COMPANION_CLIENT_ORIGINS`